### PR TITLE
About box tweaks

### PR DESCRIPTION
**Pull request by mkruselj**
* link to Surge Github page is clickable now
* tweaked the capitalization of OS and plugin types
* separated factory and user data folders to separate lines
* used full legal names of Steinberg and Apple
* removed the gray rectangle from SVGs, generated in code instead

### CODE REVIEW

File: src/common/gui/CAboutBox.cpp [hunk 1]
```diff
@@ -152,7 +160,19 @@ CAboutBox::onMouseDown(CPoint& where,
    if (!(button & kLButton))
       return kMouseEventNotHandled;
 
-   boxHide();
+   //char debug[128];
```
Comment (baconpaul) [inline src/common/gui/CAboutBox.cpp:163]: please don't leave this in the code. thanks!

File: src/common/gui/CAboutBox.cpp [hunk 2]
```diff
@@ -2,9 +2,12 @@
 #include "globals.h"
 #include "resource.h"
 #include "RuntimeFont.h"
+#include <windows.h>
```
Comment (baconpaul) [inline src/common/gui/CAboutBox.cpp:5]: bet that won't work on linux :)

Comment (mkruselj) [reply src/common/gui/CAboutBox.cpp:5]: was just for debugging, oops!

File: src/common/gui/CAboutBox.cpp (hunk 1 above)
Comment (mkruselj) [reply src/common/gui/CAboutBox.cpp:163]: oh drat, forgot to remove that.

File: src/common/gui/CAboutBox.cpp (hunk 2 above)
Comment (mkruselj) [reply src/common/gui/CAboutBox.cpp]: updated in cf82b26

File: src/common/gui/CAboutBox.cpp (hunk 1 above)
Comment (mkruselj) [reply src/common/gui/CAboutBox.cpp]: Updated in cf82b26

**Pull request merged**